src/_P043_ClkOutput.ino

```diff
@@ -41,7 +42,7 @@ boolean Plugin_043(byte function, struct EventStruct *event, String& string)
     case PLUGIN_GET_DEVICEVALUENAMES:
       {
         strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_043));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], "Temp");
+        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], "Output2");
```

Comment:
Please add it the same way as the VALUENAME1
By using PSTR macro (similar to F() macro) the string is not active in memory, but loaded from flash.
Using static strings in C++ code will have the  string content in memory twice, so that's why the flash string work-around was implemented.

Reply:
Hi, thank you for the detailed explanation, I really appreciate your support here. 

Implemented and tested